openghg/store/_populate.py

```diff
@@ -37,6 +39,7 @@ def add_noaa_obspack(
     yet e.g. aircraft, shipboard, aircorenoaa. These should be updated once
     this functionality has been added.
     """
+    from rich.progress import Progress
```

Comment:
For libraries like this I think it's best to have the import at the top of the file.

Reply:
done